### PR TITLE
Fix env vars

### PR DESCRIPTION
**Pull request by estebanmino**
**Description**

Env vars haven't been working on debug mode for a while now, this PR fixes that so we can use all the features on debug mode.

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

**Review by ibrahimtaveras00 — APPROVED** [review]
Fixes look good, QA Passed 👍

File: app/core/Analytics.js [hunk 1]
```diff
@@ -208,14 +216,15 @@ class Analytics {
...
-			console.log(`Analytics 'trackEventWithValueAndParameters' - `, event, value, info, params); // eslint-disable-line no-console
+		if (!__DEV__) {
+			RCTAnalytics.trackEvent({
+				...event,
+				...params,
+				value,
+				info
+			});
+		} else {
+			Logger.log(`Analytics 'trackEventWithValueAndParameters' - `, event, value, info, params);
```
Comment (rickycodes) [inline app/core/Analytics.js:227]: maybe this could be a method? looks like we're dong the same thing 6x

**Review by rickycodes — CHANGES_REQUESTED** [review]

**Review by andrepimenta — APPROVED** [review]
The only recommendation I have is to put code like this:
```
if (!__DEV__) {
  RCTAnalytics.trackEvent(event);
} else {
  Logger.log(`Analytics 'trackEvent' - `, event);
}
```

On a function outside on a service or helper that would look like:
```
const trackEvent = event => {
  if (!__DEV__) {
    RCTAnalytics.trackEvent(event);
  } else {
    Logger.log(`Analytics 'trackEvent' - `, event);
  }
}
```

And then you could call `trackEvent(event)` everywhere so we don't repeat code. This is especially important if we need to use that function more times in the future.

And the same for `!__DEV__ && RCTAnalytics.optIn(this.enabled);`

Otherwise looks good to me!!

**Comment by estebanmino** [comment]
@rickycodes @andrepimenta it totally makes sense, it was that way before so I didn't question it 🤦

**Review by rickycodes — APPROVED** [review]

**Pull request merged**